src/xunit.v3.runner.utility/AppDomain/AppDomainManager_NoAppDomain.cs

```diff
@@ -7,6 +8,24 @@ namespace Xunit
 {
 	class AppDomainManager_NoAppDomain : IAppDomainManager
 	{
+		public AppDomainManager_NoAppDomain(
+			string assemblyFileName,
+			string? configFileName)
+		{
+			Guard.ArgumentNotNullOrEmpty(assemblyFileName);
+
+			assemblyFileName = Path.GetFullPath(assemblyFileName);
+			Guard.FileExists(assemblyFileName);
+
+			if (configFileName == null)
+				configFileName = GetDefaultConfigFile(assemblyFileName);
+
+			if (configFileName != null)
+				configFileName = Path.GetFullPath(configFileName);
+
+			AppDomain.CurrentDomain.SetData("APP_CONFIG_FILE", configFileName);
```

Comment:
.NET 6 equivalent of this existing code:

https://github.com/xunit/xunit/blob/19fc844daabe9ae02f350340d49a4319ac6c2ed3/src/xunit.v3.runner.utility/AppDomain/AppDomainManager_AppDomain.cs#L70